### DISCUSSION TITLE
Is there a more recommended version or branch than 'develop' for an undemanding production environment?

### DISCUSSION THREAD

**Discussion by cationRI** [osTicket/osTicket#6039] [Q&A]
Hi,
 Is there a more recommended version or branch than 'develop' for an undemanding production environment?

Thanks!

**Comment by ntozier** [comment]
Stable.  You can get it at:
https://osticket.com/download

**Comment by cationRI** [comment]
Thank you. we are going to test the latest version 1.15.4.
We currently have 1.14.3

**Reply by theking81** [reply]
You should, it's far better!
